app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -434,14 +437,17 @@ function handleSizeChange(event) {
 
 function handleButtonsActivation(event) {
   var textarea = event.target;
-  var buttons = textarea.closest('.comment-form').getElementsByClassName('js-btn-enable');
-  Array.from(buttons).forEach(function(button) {
-    if (textarea.value.length > 0) {
-      button.disabled = false;
-    } else {
-      button.disabled = true;
-    }
-  });
+  var commentForm = textarea.closest('.comment-form');
```

Comment:
Food for thought, but if those buttons were in a fieldset, we could just enable/disable the fieldset. Maybe for another PR though.